cypress/api/collaborators.api.ts

```diff
@@ -9,3 +14,17 @@ export const addAdmin = (admin: string): void => {
     E2E_EMAIL_TEST_SITE.name
   )
 }
+
+export const listCollaborators = (): Cypress.Chainable<CollaboratorDto[]> => {
+  return cy.request("GET", `${BASE_URL}`).then(({ body }) => body.collaborators)
+}
+
+export const removeOtherCollaborators = (): void => {
+  listCollaborators().then((collaborators) => {
+    collaborators.forEach((collaborator) => {
+      if (collaborator.email === E2E_EMAIL_ADMIN.email) return
```

Comment:
we should remove this case - it is entirely possible that the writer of the test swapped the email to be a collab email.

we should instead rely on the guarantee that our cms guarantees >=1 collab (canot remove last admin)

Reply:
Hmm in that case, I think we might be better off allowing the user that should remain as an argument for this method?